packages/components/switch/src/Switch.tsx

```diff
@@ -9,36 +9,52 @@ import { SwitchLabel } from './SwitchLabel'
 export type SwitchProps = SwitchInputProps
 
 export const Switch = forwardRef<HTMLButtonElement, SwitchProps>(
-  ({ size = 'md', children, className, id, disabled, ...rest }, ref) => {
+  ({ size = 'md', children, className, id, disabled, reverse, ...rest }, ref) => {
```

Comment:
👋 , I had some time this morning to look at your comment. While I can set a default value of `false` for the `reverse` prop, I'm not sure if typing it as a mandatory `reverse: boolean` is a good idea.
Doing so would mean you always have to explicitly set this prop on the consumer side. wdyt?
```tsx
<Switch reverse={false | true}>Agreed</Switch>
```

Reply:
Hi, I didn't want to suggest making it a mandatory prop (it'd be quite unlike HTML and it's not a nice interface) but if the prop is typed `{ reverse?: boolean }` I would destructure it with `{ reverse = false }`, like the `size` already has in this case, so that in the component itself it's never `undefined`.

Reply:
ok, so having a default value then 👍

Reply:
That way the default value will also appear in the props table at the top of the documentation (ts-docgen will be able to read it)